### PR TITLE
update got dependency

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: packages/goto/scripts/postinstall.js [hunk 1]
```diff
@@ -9,17 +9,15 @@ const writeFile = promisify(fs.writeFile)
...
 // Lightweight `fetch` polyfill on top of `got` to allow consumption by adblocker
 const fetch = url =>
   Promise.resolve({
-    text: () => fetchText(url),
-    arrayBuffer: () => fetchBuffer(url),
-    json: () => fetchJson(url)
+    text: () =>
+      got(url)
+        .text()
+        .then(str => Buffer.from(str, 'ascii')),
```
Comment (remusao) [inline packages/goto/scripts/postinstall.js]: Why is this needed? Doesn't `got(url).text()` already resolve to a `string`?

Comment (Kikobeats) [reply packages/goto/scripts/postinstall.js]: I maintained it since you added it in the past 😄so I guess now I can remove it!

Comment (Kikobeats) [reply packages/goto/scripts/postinstall.js]: I recently discovered Node.js buffer on the browser side is ArrayBuffer, dafuq

Comment (remusao) [reply packages/goto/scripts/postinstall.js]: > I recently discovered Node.js buffer on the browser side is ArrayBuffer, dafuq

It's more `Uint8Array` which wraps around an instance of `ArrayBuffer`. I guess Node.js' `Buffer` is legacy, but at least an instance of `Buffer` is also an instance of `Uint8Array`...

Comment (remusao) [reply packages/goto/scripts/postinstall.js]: > I maintained it since you added it in the past smileso I guess now I can remove it!

I think it was needed before the update because we could not specify the expected return type from `got`! Should not be needed now.

Comment (Kikobeats) [reply packages/goto/scripts/postinstall.js]: 👌

**Comment by Kikobeats** [comment]
@remusao removed `.text()` as well and looks the `engine.bin` is created successfully 🙂

**Comment by remusao** [comment]
> @remusao removed `.text()` as well and looks the `engine.bin` is created successfully slightly_smiling_face

I think it's because the `text` part is not used, we only need json fetching and binary fetching. So you could probably just remove it... Although we never know, maybe I could change the internals :P

**Comment by Kikobeats** [comment]
then let's keep it

**Comment by remusao** [comment]
> then let's keep it

But then you need to add `.text()` :P https://github.com/microlinkhq/browserless/pull/140/commits/447829763cdf0c4412dcc47a373ad9f0c878ee6d

**Comment by Kikobeats** [comment]
I mantained `got(url)` instead of `got(url).text()` since `got`uses `text` by default as serializer serializer

**Comment by remusao** [comment]
> I mantained `got(url)` instead of `got(url).text()` since `got`uses `text` by default as serializer

That makes sense, I did not know. Thanks!
